components/event-feed-service/pkg/persistence/feed-elastic.go

```diff
@@ -510,7 +509,7 @@ func (efs ElasticFeedStore) GetActionLine(formattedFilters map[string][]string,
 		bucketSize        = strconv.Itoa(interval) + "h"
 		eventTypeItems    = "items"
 		dateHistoTag      = "dateHisto"
-		timeFormatRFC3339 = "yyyy-MM-dd'T'HH:mm:ssZ"
+		timeFormatRFC3339 = "yyyy-MM-dd'T'HH:mm:ssXXX"
```

Comment:
what is the significance of these changes? Does it going to be any breaking change on UI?

Reply:
it was not working, new formate is needed in opensearch